lib/tmdb/client.rb

```diff
@@ -11,7 +11,7 @@ def request(endpoint, params)
         return { results: [] } if params[:query].present? && searchable_query(params[:query]).empty?
 
         api_path = build_url(endpoint, params)
-        response = open("#{BASE_URL}#{api_path}").read
+        response = URI.open("#{BASE_URL}#{api_path}").read
```

Comment:
How in the heck did this even work before?

Reply:
🤷 Based on the deprecation warning, I think it was still calling `open` on `URI`